### PR TITLE
Top-level suggestions

### PR DESCRIPTION
**Pull request by gewarren**
@BillWagner - these are some things I noticed. Feel free to accept or decline.

### CODE REVIEW

File: docs/core/tutorials/top-level-templates.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ These two forms represent the same program. Both are valid with C# 10.0. When yo
 
 ## Use the new program style
 
-The features that make the new program simpler are top-level statements, global usings, and implicit usings.
+The features that make the new program simpler are *top-level statements* and *global using directives*.
```
Comment (tdykstra) [inline docs/core/tutorials/top-level-templates.md:41]:
```suggestion
The features that make the new program simpler are *top-level statements* and *global `using` directives*.
```

File: docs/core/tutorials/top-level-templates.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ These two forms represent the same program. Both are valid with C# 10.0. When yo
...
 
 [Top-level statements](../../csharp/fundamentals/program-structure/top-level-statements.md) means the compiler generates the namespace, class, and method elements for your main program. You can look at the code for the new application and imagine that it contains the statements inside the `Main` method generated by earlier templates. You can add more statements to the program, just like you can add more statements to your `Main` method in the traditional style. You can even add functions. They're created as local functions nested inside the generated `Main` method.
 
-*Global usings* means the compiler automatically imports a set of [`using` directives](../../csharp/language-reference/keywords/using-directive.md) based on the project type. For console applications, the following directives are implicitly included in every source file in the application:
+*Global using directives* means the compiler automatically imports a set of [`using` directives](../../csharp/language-reference/keywords/using-directive.md) based on the project type. For console applications, the following directives are implicitly included in every source file in the application:
```
Comment (tdykstra) [inline docs/core/tutorials/top-level-templates.md:45]:
```suggestion
*Global `using` directives* means the compiler automatically imports a set of [`using` directives](../../csharp/language-reference/keywords/using-directive.md) based on the project type. For console applications, the following directives are implicitly included in every source file in the application:
```

**Review by tdykstra — COMMENTED** [review]
A couple of nit suggestions.

File: docs/core/tutorials/top-level-templates.md (hunk 1 above)
Comment (BillWagner) [inline docs/core/tutorials/top-level-templates.md:41]: We should keep "implicit using directives" here. (I haven't written the two articles I'll reference here yet, but there is a difference. "global usings" are the statements you add. "implicit using directives" are a set of global usings that MSBuild will create using a source generator).

```suggestion
The features that make the new program simpler are *top-level statements*, *global using directives*, and *implicit using directives*.
```

**Review by BillWagner — APPROVED** [review]
Thanks @gewarren these are great changes.

I had one edit, which is a clarification for a couple articles I haven't written yet. Then, this is ready to :shipit:

**Pull request merged**